inc/language-utils.php

```diff
@@ -19,8 +19,13 @@ class WPSEO_Language_Utils {
 	public static function get_language( $locale = null ) {
 		$language = 'en';
 
-		if ( ! empty( $locale ) && strlen( $locale ) >= 2 ) {
-			$language = substr( $locale, 0, 2 );
+		if ( ! empty( $locale ) && is_string( $locale ) ) {
```

Comment:
By negating this conditional check, an early return with the default is possible.
This will reduce the complexity of the function, as we would not have to indent the code inside the conditional anymore.